ush/wave_tar.sh

```diff
@@ -32,40 +32,39 @@
 
   alertName=$(echo $RUN|tr [a-z] [A-Z])
 
-  set +x
-  echo ' '
-  echo '+--------------------------------+'
-  echo '!         Make tar file          |'
-  echo '+--------------------------------+'
-  echo "   ID              : $1"
-  echo "   Type            : $2"
-  echo "   Number of files : $3"
-  set_trace
-
 
 # 0.b Check if type set
 
   if [[ "$#" -lt '3' ]]
   then
-    set +x
-    echo ' '
-    echo '********************************************'
-    echo '*** VARIABLES IN ww3_tar.sh NOT SET ***'
-    echo '********************************************'
-    echo ' '
-    set_trace
+    echo 'ERROR: VARIABLES IN ww3_tar.sh NOT SET'
```

Comment:
FATAL ERROR or WARNING?

Reply:
Error as it's exiting with non-zero status.